st2common/st2common/rbac/types.py

```diff
@@ -38,8 +38,8 @@ class PermissionType(Enum):
     PACK_ALL = 'pack_all'
 
     # Note: Right now we only have read endpoints for sensors types
-    SENSOR_VIEW = 'sensor_view'
-    SENSOR_ALL = 'sensor_all'
+    SENSOR_VIEW = 'sensor_type_view'
```

Comment:
`def get_resource_type(cls, permission_type):` Will not work with the way it is right now if you use `sensor_*` instead of `sensor_type_*`.

I am not sure how you would want me to resolve the discrepancy.